src/components/TutorialsOverview/Hero.vue

```diff
@@ -17,13 +17,13 @@
         <TimerIcon />
         <span class="meta-content">
           <strong class="time">{{estimatedTime}}</strong>
```

Comment:
![image](https://user-images.githubusercontent.com/87735557/218894168-c985ed8e-7d18-4f0b-aaad-0769dbeab4b0.png)
`min` was not localized here

Reply:
This value comes from the backend at `topic.estimatedTime` so it's not part of this PR